tests/summarizer/test_openai_summarizer.py

```
@@ -0,0 +1,315 @@
+    @patch('pod_tenuki.summarizer.openai_summarizer.cost_tracker')
+    def test_generate_summary_custom_model(self, mock_cost_tracker, mock_openai_class, mock_openai_response):
+        """Test generating summary with custom model."""
+        # Setup mock client
+        mock_client = Mock()
+        mock_client.chat.completions.create.return_value = mock_openai_response
+        mock_openai_class.return_value = mock_client
+
+        summarizer = OpenAISummarizer(api_key="test-api-key")
+
+        # Generate summary with custom model
+        title, description, topics = summarizer.generate_summary(
```

Ruff (0.14.5)

Unpacked variable `title` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)